test/bin/gh-pages.spec.js

```diff
@@ -88,36 +88,29 @@ describe('gh-pages', () => {
         error:
           'Could not parse name and email from user option "junk email" (format should be "Your Name <email@example.com>")',
       },
-      {
-        args: ['.'],
-        error:
-          'No base directory specified. The `--dist` option must be specified.',
-      },
```

Comment:
I removed this test case because commander exits when the `--dist` option is not provided.